### ISSUE TITLE
implement NEW_TURN thunk

### ISSUE THREAD

**Issue by Bebersohl** [Bebersohl/hp-redux#52]
We need an action to start a players turn. This thunk should:
* Toggle `activePlayer` property in `game`
* Gain 1 mana
* Draw 1 card
* Reset minion properties:
  * `frozenFor` decremented
  * `exhausted`, `alreadyAttacked`, `windFuryUsed` set to false
* Reset hero properties: `alreadyAttacked`, `windFuryUsed`

We need a general action to toggle any boolean on a character. Ill write a new issue for this that should be completed first
